examples/fractal.py

```diff
@@ -3,12 +3,12 @@
 ti.init(arch=ti.gpu)
 
 n = 320
-pixels = ti.var(dt=ti.f32, shape=(n * 2, n))
+pixels = ti.field(float, (n * 2, n))
```

Comment:
WDYT? I'll update doc for this 2pvc #1576.

Reply:
What does "2pvc" mean? Sorry but I don't understand...